build/xtask/src/config.rs

```diff
@@ -780,7 +780,7 @@ impl BuildConfig<'_> {
 
         cmd.arg(
             "-Zallow-features=asm_sym,asm_const,named-profiles,naked_functions,\
-cmse_nonsecure_entry,array_methods,backtrace,proc_macro_span,const_trait_impl,\
+cmse_nonsecure_entry,array_methods,backtrace,proc_macro_span,proc_macro_span_shrink,const_trait_impl,\
```

Comment:
What prompted this addition?

Reply:
proc-macro2 now uses this feature, without having bumped their major version. Multiple Hubris crates make use of that crate, but only specify the version as = "1", which is unfortunate.